doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3683,11 +3683,7 @@ The general command to add a hint to some databases :n:`{+ @ident}` is
       Local is useless since hints do not survive anyway to the closure of
       sections.
 
-   .. cmdv:: Local Hint @hint_definition
-
-      Idem for the core database.
```

Comment:
Any reason to remove this?  Sorry if I missed something in the discussion

Reply:
If this is deprecated, we should probably still document it, and mention that it's deprecated, shouldn't we ?

Reply:
The main variant documents the option of not putting the database explicitly and says that this is deprecated.